### PR TITLE
feat: allowOwnerOffCurve flag was added

### PR DESCRIPTION
**Pull request by tjm-lido**

### CODE REVIEW

File: docs/frontend-integration/sdk/sdk-methods.mdx [hunk 1]
```diff
@@ -34,13 +34,21 @@ const { transactionHash, stSolAccountAddress } = await solidoSDK.stake({
...
+    allowOwnerOffCurve: true,
+});
 ```
 
 **Parameters:**
 
 - *`amount`* - The amount of SOL-s which need to stake
 - *`wallet`* - Wallet instance from [@solana/wallet-adapter-base](https://www.npmjs.com/package/@solana/wallet-adapter-base)
 - *`setTxStage?`* - Optional callback for getting information about transaction stage
+- *`allowOwnerOffCurve?` - Allow the owner account to be a PDA (Program Derived Address), when we are creating ATA (Associated Token Account)
```
Comment (tataklim) [inline docs/frontend-integration/sdk/sdk-methods.mdx]: Maybe we can add here for terms links to Solana docs to simplify users' experience?

File: docs/frontend-integration/sdk/sdk-methods.mdx [hunk 2]
```diff
@@ -34,13 +34,21 @@ const { transactionHash, stSolAccountAddress } = await solidoSDK.stake({
     amount: 20,
     wallet: walletInstance,
 });
+
+// For PDA accounts
```
Comment (tataklim) [inline docs/frontend-integration/sdk/sdk-methods.mdx]: I think it's incorrect to write `PDA accounts`
Like PDA is a type of addresses
As Solana cookbook says `PDAs are home to accounts...`
Maybe let it just like `for PDAs (Program Derived Addresses)`?

**Review by rgb2hsl — APPROVED** [review]

File: docs/frontend-integration/sdk/sdk-methods.mdx (hunk 1 above)
Comment (tjm-lido) [reply docs/frontend-integration/sdk/sdk-methods.mdx]: done

File: docs/frontend-integration/sdk/sdk-methods.mdx (hunk 2 above)
Comment (tjm-lido) [reply docs/frontend-integration/sdk/sdk-methods.mdx]: done

**Pull request merged**